configcatclient/__init__.py

```diff
@@ -1,19 +1,21 @@
 from .configcatclient import ConfigCatClient
 from .interfaces import ConfigCatClientException
+from .datagovernance import DataGovernance
 
 
-def create_client(sdk_key):
+def create_client(sdk_key, data_governance=DataGovernance.Global):
     """
     Create an instance of ConfigCatClient and setup Auto Poll mode with default options
 
     :param sdk_key: ConfigCat SDK Key to access your configuration.
+    :param data_governance:
```

Comment:
TODO: description

Reply:
Description: Set this parameter to restrict the location of your feature flag and setting data within the ConfigCat CDN. Default: Global, This parameter must be in sync with the preferences on: https://app.configcat.com/organization/data-governance

EU enum: Your data will be published to CDN nodes only in the EU.
Global enum: Your data will be published to all ConfigCat CDN nodes to guarantee lowest response times.